schema/cam_schema_1-1-2.json

```diff
@@ -31,7 +31,7 @@
     "version": {
       "type": "string",
       "description": "json message format version",
-      "const": "1.1.1"
+      "const": "1.1.2"
```

Comment:
The version is also set in the file's name  
Which is not necessarily a good thing as we would mostly lose changes history over versions